code/modules/vore/eating/simple_animal_vr.dm

```diff
@@ -75,3 +75,27 @@ mob/living/simple_animal/custom_emote()
 mob/living/simple_animal/say()
 	if (away_from_players()) return
 	. = ..()
+
+/mob/living/simple_animal/attackby(var/obj/item/O, var/mob/user)
+	if (istype(O, /obj/item/weapon/newspaper) && !(ckey || (hostile && faction != user.faction)) && isturf(user.loc))
+		if (retaliate && prob(vore_pounce_chance/2)) // This is a gamble!
+			user.Weaken(5) //They get tackled anyway whether they're edible or not.
+			user.visible_message("<span class='danger'>\the [user] swats \the [src] with \the [O] and promptly gets tackled!</span>!")
+			if (will_eat(user))
+				stop_automated_movement = 1
+				animal_nom(user)
+				update_icon()
+				stop_automated_movement = 0
+		else
+			user.visible_message("<span class='info'>\the [user] swats \the [src] with \the [O]!</span>!")
+			for(var/I in vore_organs)
+				var/datum/belly/B = vore_organs[I]
+				B.release_all_contents(include_absorbed = TRUE) // Until we can get a mob version of unsorbitol or whatever, release absorbed too
+			for(var/mob/living/L in range(1)) //add everyone on the tile to the do-not-eat list for a while
```

Comment:
range(1) is everyone within a 3x3 area. It would also add the newspaper user, among anyone else standing nearby to excludes. range(0) is same turf. Also range includes things like INVISIBLE PEOPLE which is nonsensical. You could do oviewers(0) which would be 'my turf, not me, only mobs' and doesn't require the type check there. There's a helper proc for this basically, which is living_mobs(range)

Reply:
disabling vore_active doesn't make will_eat return false, which meant runtime was being all like "okay, I WILL EAT that person so they're a valid target. But I am not VORE ACTIVE so I will attack them instead of eating them."

... I could just add vore_active to the will_eat check.